spec/cyclonedx/cocoapods/bom_builder_spec.rb

```diff
@@ -348,13 +363,35 @@
           components_generated_from_bom_builder = xml.at('bom/components')
 
           components = Nokogiri::XML(Nokogiri::XML::Builder.new(encoding: 'UTF-8') do |xml|
-            xml.components('xmlns': described_class::NAMESPACE) do
+            xml.components(xmlns: described_class::NAMESPACE) do
               bom_builder.pods.each { |pod| pod.add_to_bom(xml) }
             end
           end.to_xml).at('components')
 
           expect(components_generated_from_bom_builder).to be_equivalent_to(components)
         end
+
+        it 'should generate a child dependencies node' do
+          expect(xml.at('bom/dependencies')).not_to be_nil
+        end
+
+        it 'shoudl properly set dependencies node' do
+          dependencies_generated_from_bom_builder = xml.at('bom/dependencies')
+
+          dependencies = Nokogiri::XML(Nokogiri::XML::Builder.new(encoding: 'UTF-8') do |xml|
```

Comment:
This look like it's just copied the dependency builder out of bom_builder.rb and placed it here.  This doesn't test the code, but simply tests that you copy/pasted it correctly.

A better test would be something that has hardcoded dependencies as a string here and compare the generated xml to that.  Something like comparing to this string:

```ruby
'<dependencies><dependency ref="pkg:cocoapods/Alamofire@5.6.2"/><dependency ref="pkg:cocoapods/MSAL@1.2.1"><dependency ref="pkg:cocoapods/MSAL@1.2.1#app-lib"/></dependency><dependency ref="pkg:cocoapods/FirebaseAnalytics@7.10.0#app-lib"/><dependency ref="pkg:cocoapods/RxSwift@5.1.2#app-lib"/><dependency ref="pkg:cocoapods/Realm@5.5.1#app-lib"/></dependencies>'
```

Reply:
I tough the same let me replace it

Reply:
well you cannot use it like this because this tests will run twice, one for component without pods and the other with pods... so the first one is just 
`<dependencies/>`

Reply:
Added the xml strings directly on the tests.